utils/testrunner/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@pptr/testrunner",
```

Comment:
let's remove package.json - we don't publish it atm

Reply:
Done.